src/less/main.less

```diff
@@ -1852,6 +2044,14 @@ span.day {
   }
 }
 
+.deployment-phases-list > .column {
```

Comment:
... after merging this part was also gone so my addition to the styling would be:
```
.deployment-phases-list {
  overflow: auto;
  max-width: 900px;
  & > .column {
    min-width: 115px;
    margin: 0 5px;
    justify-content: flex-end;
    & > div {
      margin: 5px 0;
    }
    .MuiChip-root {
      width: fit-content;
    }
  }
}
```
or a mix of both...

Reply:
Added in b0050eb